src/menu/Console.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "core/bridge/consolevariablebridge.h"
 #include "ImGuiImpl.h"
-#include <ImGui/imgui.h>
```

Comment:
Why are we removing the ImGui path from the header include here?

Reply:
I put the FetchContent statement in my project's CMakeLists.txt to my fork of LUS to test and it was erroring out because it couldn't find ImGui/imgui.h. Removing ImGui from that path fixed the error.